modules/cms/tests/classes/RouterTest.php

```diff
@@ -1,13 +1,18 @@
 <?php
 
+namespace Cms\Tests\Classes;
+
+use System\Tests\Bootstrap\TestCase;
 use Cms\Classes\Router;
 use Cms\Classes\Theme;
+use ReflectionClass;
+use function count;
```

Comment:
Another PHPStorm artifact?

Reply:
As above

Reply:
Although I prefer to `use ReflectionClass;` over `new \ReflectionClass();`. Your call though, not massively bothered.